### PR TITLE
Add info about input data and concat script

### PR DESCRIPTION
**Pull request by darafferty**
This PR adds more information to the documentation about the requirements of the input data and adds a script for concatenating the MS files output by Prefactor for use in Rapthor.

### CODE REVIEW

File: bin/concat_prefactor_files [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    Concatenate MS files from Prefactor for input to Rapthor
+
+    Parameters
+    ----------
+    input_path : str
+        Full path to the directory with the Prefactor MS files
+    output_file : str
+        Filename of output file
+    """
+    msfiles = glob.glob(os.path.join(input_path, '*.ms'))
```
Comment (gmloose) [inline bin/concat_prefactor_files:26]: I think MS-files could have both `.ms` and `.MS` as extension. Maybe good to check for both?

File: bin/concat_prefactor_files [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            file_bandwidth = sw.col('TOTAL_BANDWIDTH')[0]
+            nchans = sw.col('CHAN_WIDTH')[0].shape[0]
+            chwidth = sw.col('CHAN_WIDTH')[0][0]
+            first = False
+        else:
+            assert file_bandwidth == sw.col('TOTAL_BANDWIDTH')[0]
+            assert nchans == sw.col('CHAN_WIDTH')[0].shape[0]
+            assert chwidth == sw.col('CHAN_WIDTH')[0][0]
+        freqs.append(freq)
+        sw.close()
```
Comment (gmloose) [inline bin/concat_prefactor_files:46]: Not sure how important this is, but the table will not be closed when an exception is raised; e.g. when an `assert` is triggered.

File: bin/plotrapthor [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/env python
```
Comment (gmloose) [inline bin/plotrapthor:1]: Shouldn't this be `python3`?

File: bin/concat_prefactor_files [hunk 4]
```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
```
Comment (gmloose) [inline bin/concat_prefactor_files:1]: Shouldn't this be `python3`?

File: bin/concat_prefactor_files (hunk 1 above)
Comment (darafferty) [reply bin/concat_prefactor_files:26]: Yes, good idea. I'll add a check for `.MS`.

File: bin/concat_prefactor_files (hunk 2 above)
Comment (darafferty) [reply bin/concat_prefactor_files:46]: Well spotted -- I will move the close() before the checks to be sure.

File: bin/plotrapthor (hunk 3 above)
Comment (darafferty) [reply bin/plotrapthor:1]: I changed it to match `bin/rapthor` and the Prefactor scripts, but after researching it a bit I think you're right and it should be `python3`

Comment (darafferty) [reply bin/plotrapthor:1]: And is `/usr/bin/env python3` OK, or is `/usr/bin/python3` better?

Comment (gmloose) [reply bin/plotrapthor:1]: The former is the right way, AFAIK.

Comment (darafferty) [reply bin/plotrapthor]: OK, thanks. I updated all the Python scripts to make the shebang usage uniform throughout.

**Pull request merged**